### PR TITLE
Create a calculator component

### PR DESCRIPTION
**Pull request by AwaMelvine**

### CODE REVIEW

File: Project-II-Calculator/calculator/src/components/ButtonContainers/NumberButtonsContainer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const NumberButtonsContainer = ({ handleClick }) => (
+  <div className="number-buttons-container">
+    <ActionButton
+      customClassName="clear"
+      label={data.clear}
+      handleClick={handleClick}
+    />
+    {data.numbers.map(number => (
+      <NumberButton key={number} label={number} handleClick={handleClick} />
+    ))}
```
Comment (mattshardman) [inline Project-II-Calculator/calculator/src/components/ButtonContainers/NumberButtonsContainer.js:19]: Good use of ```.map``` to keep things DRY

File: Project-II-Calculator/calculator/src/App.js [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0
+    };
+  }
+  handleClick = value => {
+    const { total } = this.state;
+    if (value === "clear") {
+      this.setState({ ...this.state, total: 0 });
```
Comment (mattshardman) [inline Project-II-Calculator/calculator/src/App.js:16]: You only need to spread the ```this.state``` object if there are other properties in the object

**Comment by mattshardman** [comment]
Really good job Melvine. 

- [x]  Your calculator project closely resembles the design files
- [x]  Your git work flow is fluid and descriptive
- [x]  Your have separated the buttons and screen into separate components logically
- [x]  You have gone above the MVP by adding the ability to update the screen

**Possible things to work on**
- couple of minor comments but nothing major, good job

Overall really good job, everything is implemented correctly and you've added some functionality.

File: Project-II-Calculator/calculator/src/App.js (hunk 2 above)
Comment (AwaMelvine) [reply Project-II-Calculator/calculator/src/App.js:16]: Oh okay, thanks. Noted!

**Pull request merged**